examples/hello_world/client/nodejs/index.js

```diff
@@ -17,31 +17,24 @@
 const fs = require('fs');
 const protobufjs = require('protobufjs');
 const grpc = require('@grpc/grpc-js');
-const grpcProtoLoader = require('@grpc/proto-loader');
 
-const CERT_PATH = __dirname + '/../../../certs/local/ca.pem';
-const SERVICE_PROTO_PATH = __dirname + '/../../proto/hello_world.proto';
-const OAK_LABEL_PROTO_PATH =
-  __dirname + '/../../../../oak_abi/proto/label.proto';
+const REPOSITORY_ROOT = `${__dirname}/../../../..`;
+const CERT_PATH = `${REPOSITORY_ROOT}/examples/certs/local/ca.pem`;
+const OAK_LABEL_PROTO_PATH = `${REPOSITORY_ROOT}/oak_abi/proto/label.proto`;
+const SERVICE_PROTO_PATH = `${REPOSITORY_ROOT}/examples/hello_world/proto/hello_world.proto`;
 
 // Keep in sync with /oak_runtime/src/node/grpc/server/mod.rs.
 const oakLabelGrpcMetadataKey = 'x-oak-label-bin';
 
 async function main() {
-  const [oaklabelDefinition, helloWorldDefinition] = await Promise.all([
-    // We use two different libs with a similar APIs to load proto files,
-    // which due to the design of the `@grpc/grpc-js` lib.
-    // `@grpc/proto-loader` is a wrapper around `protobufjs` that adds
-    // functionality required for working with `grpc-js`, but omits others.
-    // Hence we use grpcProtoLoader for gRPC services, and protobufjs
-    // for all other protos.
-    protobufjs.load(OAK_LABEL_PROTO_PATH),
-    grpcProtoLoader.load(SERVICE_PROTO_PATH),
```

Comment:
Stumbled upon this when revisiting this code to add challenge-response auth. Imho this makes things quite confusing, so I really wanted to fix it.

Reply:
Unrelated to this PR, but imho the architecture of `@grpc/proto-loader` should probably be changed to just export the underlying helper functions, instead of needlessly wrapping a whole protobufjs instance. That is if the abstraction is even needed, which imho it probably isn't.